testing/tf_job_simple_test.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+
+# Copyright 2018 The Kubeflow Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Test tf_job_simple prototype. It creates a component from the tf_job_simple
+prototype and applies it to the k8s cluster. It then verifies that two pods
+and services with the appropriate label get created.
+"""
+
+import argparse
+import logging
+import os
+
+from kubeflow.testing import test_helper, util
+from retrying import retry
+
+
+def parse_args():
+  parser = argparse.ArgumentParser()
+  parser.add_argument(
+    "--src_dir",
+    default="",
+    type=str,
+    help="The kubeflow src directory")
+  args, _ = parser.parse_known_args()
+  return args
+
+@retry(wait_fixed=5000, stop_max_attempt_number=20)
+def wait_for_tf_job():
+  out = util.run(["kubectl", "get", "pods", "-l",
+                  "tf_job_name=mycnnjob", "-ndefault"])
```

Comment:
If namespace is default, do we need to delete them? I guess jobs will gone after finish, how about the service?

Reply:
We will be deleting the cluster at the end so I did not bother deleting this.

Reply:
Are we using kubeflow-testing cluster in kubeflow-ci?

Reply:
No, we'll be creating a new cluster for every test